dvc/render/match.py

```diff
@@ -43,17 +44,20 @@ def group_definitions(self):
 
     def get_definition_data(self, target_files, rev):
         result = {}
-        for file in target_files:
+        for definition_file in target_files:
+            if os.name == "nt":
+                source_file = _normpath(definition_file).replace("\\", "/")
+            else:
+                source_file = definition_file
```

Comment:
This just fixes the #8689 but all the path handling logic inside `plots.show` feels like:

![blinds-venetian](https://user-images.githubusercontent.com/12677733/222200788-f796a944-774e-41e9-a400-c1d624b1cce8.gif)

I gave it a try but there are quite a few changes needed so leaving that for a potential follow-up wjile making this small fix on the UI level.